sys/include/fido2/ctap/ctap_mem.h

```diff
@@ -142,21 +132,15 @@ ctap_status_code_t fido2_ctap_mem_read_state_from_flash(ctap_state_t *state);
 ctap_status_code_t fido2_ctap_mem_write_state_to_flash(ctap_state_t *state);
 
 /**
- * @brief Find resident credential for @p rp_id_hash in flash
- *
- * The function stores the flash address of the next credential in @p addr.
- * This allows for consecutive calls of the function in order to find all
- * stored credentials stored for the relying party identified by
- * @p rp_id_hash.
+ * @brief Iterates over all resident credentials in flash memory
  *
- * @param[in]  key       pointer to authenticator state
- * @param[in]   rp_id_hash pointer to hash of rp domain string
- * @param[in] addr pointer to address where to read from
+ * @param[in] key           pointer to authenticator state
+ * @param[in, out] state    iteration state of resident credential. Must point to a NULL
```

Comment:
```suggestion
 * @param[out] key           pointer to authenticator state
 * @param[inout] state    iteration state of resident credential. Must point to a NULL
```

also it is a bit confusing to the reader that `key` denotes an authenticator state